pinot-spi/src/main/java/org/apache/pinot/spi/config/instance/InstanceDataManagerConfig.java

```diff
@@ -68,4 +68,10 @@ public interface InstanceDataManagerConfig {
   long getExternalViewDroppedMaxWaitMs();
 
   long getExternalViewDroppedCheckIntervalMs();
+
+  String getUpsertDefaultMetadataManagerClass();
```

Comment:
how about add `Map<String, String> getUpsertConfigs()` here and return _instanceDataManagerConfiguration.subset("upsert") in HelixInstanceDataManagerConfig.java

so that we can keep those upsert specific configs to be defined inside the BaseTableUpsertMetadataManager, who should know what upsert specific configs it needs.